### PR TITLE
[201911][Monit] Unmonitor processes in disabled containers

### PR DESCRIPTION
**Pull request by yozhao101**
- Why I did it
We want to let Monit to unmonitor the processes in containers which are disabled in FEATURE table such that
Monit will not generate false alerting messages into the syslog.

- How I did it
Monit will periodically run a script which accepts three parameters: <container_name>, <process_name> and
<process_cmdline>. This script will first check whether the container is disabled in the FEATURE table or not.
If it is disabled, Monit will skip monitoring the processes. Otherwise, this script will leverage psutil library to inspect
the process tree in host to look for the processes. If the process is not found, then an alerting message will be written
into syslog.

- How to verify it
We can change the state field of a container in FEATURE table from enabled to disabled and then kill a critical
process in it to see whether Monit can generate the alerting message in syslog or not. The message format in syslog is:
<process_name> is not running.

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by jleveque** [comment]
There are multiple conflicts. Can you fix?

**Comment by yozhao101** [comment]
I am fixing the conflicts.

**Comment by yozhao101** [comment]
> There are multiple conflicts. Can you fix?

Fixed.

File: files/build_templates/sonic_debian_extension.j2 [hunk 1]
```diff
@@ -109,6 +109,18 @@ sudo rm -rf $FILESYSTEM_ROOT/$REDIS_DUMP_LOAD_PY2_WHEEL_NAME
...
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install psutil
+
+# Install SwSS SDK Python 3 package
+# Note: the scripts will be overwritten by corresponding Python 2 package
+if [ -e {{swsssdk_py3_wheel_path}} ]; then
+    SWSSSDK_PY3_WHEEL_NAME=$(basename {{swsssdk_py3_wheel_path}})
+    sudo cp {{swsssdk_py3_wheel_path}} $FILESYSTEM_ROOT/$SWSSSDK_PY3_WHEEL_NAME
+    sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install $SWSSSDK_PY3_WHEEL_NAME
+    sudo rm -rf $FILESYSTEM_ROOT/$SWSSSDK_PY3_WHEEL_NAME
+fi
```
Comment (jleveque) [inline files/build_templates/sonic_debian_extension.j2:122]: Why did you add this?

Comment (yozhao101) [reply files/build_templates/sonic_debian_extension.j2:122]: That's a mistake. I think this happened when I fixed the conflicts during the cherry-pick. Let me fix it.

**Comment by yozhao101** [comment]
I introduced other dirty commits in the last commit of this PR during cherry-pick. This PR will be closed and I will open a new one.
